packages/cli/lib/project-generator.js

```diff
@@ -11,6 +11,7 @@ module.exports = class ProjectGenerator extends BaseGenerator {
   // Note: arguments and options should be defined in the constructor.
```

Comment:
Add JSDocs please. I know they weren't here before, but let's fix that so that people know what the buildOptions do/are for.

Reply:
I'm not sure how to add JSDocs for `buildOptions` but I've added a comment explaining what `buildOptions` is for.